piikii-output-web/lemon/src/main/kotlin/com/piikii/output/web/lemon/config/LemonConfig.kt

```diff
@@ -15,11 +15,21 @@ class LemonConfig {
             .baseUrl(lemonProperties.url.api)
             .build()
     }
+
+    @Bean
+    fun lemonCoordinateApiClient(lemonProperties: LemonProperties): RestClient {
+        return RestClient.builder()
+            .defaultHeader("Authorization", lemonProperties.apiKey)
```

Comment:
```suggestion
            .defaultHeader(HttpHeaders.AUTHORIZATION, lemonProperties.apiKey)
```
위처럼 상수를 사용해도 됩니다 ㅎㅎ (뭐 어차피 빈 코드라 상관은 없겠지만)

Reply:
조습니다 반영완료!